11_notify/n0npax/pkg/puppy/lostService.go

```diff
@@ -0,0 +1,31 @@
+package puppy
+
+import (
+	"net/http"
+
+	middleware "github.com/anz-bank/go-course/11_notify/n0npax/pkg/middleware"
+	"github.com/gin-gonic/gin"
+)
+
+var SlowRequestDuration = "2s"
```

Comment:
In line with the other comment, This should be a time.Duration
```go
var SlowRequestDuration = 2 * time.Second
```

That way you don't have to worry about the input being invalid.

Reply:
fixed. Thanks!